Algolia.Search/Index.cs

```diff
@@ -590,9 +590,13 @@ public Task<JObject> BrowseFromAsync(Query q, string cursor, RequestOptions requ
             string cursorParam = "";
             if (cursor != null && cursor.Length > 0)
             {
-                cursorParam = string.Format("&cursor={0}", WebUtility.UrlEncode(cursor));
+                Dictionary<string, object> body = new Dictionary<string, object>();
+                body["cursor"] = cursor;
+
+                return _client.ExecuteRequest(AlgoliaClient.callType.Read, "POST", string.Format("/1/indexes/{0}/browse?{1}", _urlIndexName, q.GetQueryString()), body, token, requestOptions);

```

Comment:
It's also a nit but it would be more "cleaner" to use a string interpolation 
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Reply:
I used this one because it's how it's done in the client. But it's good to know that there is an easy way to do string interpolation.